quipucords/api/deployments_report/view.py

```diff
@@ -64,27 +64,41 @@ def deployments(request, pk=None):
             'report_id': [_(messages.COMMON_ID_INV)]
         }
         raise ValidationError(error)
-
+    mask_report = request.query_params.get('mask', False)
     report = get_object_or_404(DeploymentsReport.objects.all(), report_id=pk)
     if report.status != DeploymentsReport.STATUS_COMPLETE:
         return Response({'detail':
                          'Deployment report %s could not be created.'
                          '  See server logs.' % report.details_report.id},
                         status=status.HTTP_424_FAILED_DEPENDENCY)
 
-    return Response(build_cached_json_report(report))
+    return Response(build_cached_json_report(report, mask_report))
 
 
-def build_cached_json_report(report):
+def build_cached_json_report(report, mask_report):
```

Comment:
We may want to add `mask_report` to the doc string just to clarify that the mask_report is a boolean and not the actual mask report.

Reply:
Done.